schema/google/showcase/v1beta1/echo.proto

```diff
@@ -106,9 +106,19 @@ service Echo {
   };
 }
 
-// The request message used for the Echo, Collect and Chat methods. If content
-// is set in this message then the request will succeed. If status is set in 
-// this message then the status will be returned as an error.
+// A throwaway enum used to test enum capabilities in GAPIC surfaces
+enum Throwaway {
```

Comment:
Can we think of a better name than `Throwaway`? To make things fit in better and form some semblance of a "real" API (I realize it's not).

Alternatively, `identity.proto` has more "context" built into it that might provide a better spot to implement an enum.

Reply:
Agreed, 'Throwaway' was my throwaway name idea :P
My initial thought was that a quick enum check fit more in with echo than identity, but I'll take another look through identity and reevaluate my opinion.

Reply:
Went with 'severity' instead of 'throwaway'

Reply:
Thanks, sgtm